nettrine/app/Model/Database/Entity/Category.php

```diff
@@ -1,4 +1,6 @@
-<?php declare(strict_types = 1);
+<?php
+
+declare(strict_types=1);
```

Comment:
Na jeden radek prosim

Reply:
Projdi i ty dalsi pripady, at to nemusim vsechno komentovat

Reply:
hotovo